pages/orders/[orderId]/labels.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+  Button,
+  Checkbox,
+  CheckboxLabel,
+  Form,
+  FormGroup,
+  Grid,
+  GridItem,
+  Input,
+  Select,
+  Text,
+} from '@bigcommerce/big-design';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+const StyledGrid = styled(Grid)`
+  grid-template: 'form print' auto / 1fr auto;
+
+  > .bd-button {
+    position: sticky;
+    top: 0;
+  }
+`;
+
+const OrderLabelPage: React.FC = () => {
+  const [shipping, setShipping] = useState('other');
+  const [shippingType, setShippingType] = useState<string>();
+  const [checked, setChecked] = useState(false);
+
+  const toggleChecked = () => {
+    setChecked(!checked);
```

Comment:
if the next state depends on the previous, we should prefer to use functional setState, in a similar fashion to setting functional updates in classes.  

Please see:
https://reactjs.org/docs/hooks-reference.html#functional-updates
https://stackoverflow.com/questions/55510565/do-we-still-need-functional-setstate-way-in-react-hooks#answer-55510736

```suggestion
    setChecked(prevChecked => !prevChecked);
```

Reply:
This seems more complicated to me lol.

Reply:
I got rid of this page... it added no value to the PR.

Reply:
@jorgemoya it has to do with how React batches calls to setState, which unfortunately React has done since class based components.  In other words, React doesn't handle multiple calls to state synchronously - the best way to guarantee future state that depends on the previous state (e.g. a counter or toggle) is to use functional updates. The stackoverflow link gives a pretty good example.

basically:
```jsx
const [count, setCount] = useState(1);

const handleClick = () => {
  setCount(count + 1);
  setCount(count + 1);
  setCount(count + 1);
}; 
// results in count === 2
```
This could also happen if the handleClick had just one `setCount`, but handleClick was hit multiple times.